superset/assets/package.json

```diff
@@ -68,6 +68,7 @@
     "geojson-extent": "^0.3.2",
     "geolib": "^2.0.24",
     "immutable": "^3.8.2",
+    "is-react": "^1.1.1",
```

Comment:
@betodealmeida @mistercrunch was it necessary to add this dependency for functionality that is already provided by React natively? 

from the [source](https://github.com/treyhuffine/is-react/blob/master/index.js#L32): 
`React.isValidElement(typeElement)`

Reply:
FYI this added react 16 as a direct dependency (not a peer dependency), which means we have 2 versions of react floating around. it'd be great to get rid of this.

Reply:
Let me take a look.

Reply:
Fixed in https://github.com/apache/incubator-superset/pull/5478.